src/constants/chains.ts

```diff
@@ -35,6 +35,17 @@ export const chainInfo: { [chainId in ChainId]: ChainInfo } = {
     rewardsSubgraphUrl: "https://api.thegraph.com/subgraphs/name/0xjurassicpunk/looks-distribution",
     cloudinaryUrl: "https://looksrare.mo.cloudinary.net/goerli",
   },
+  [ChainId.SEPOLIA]: {
+    label: "Sepolia",
+    appUrl: "https://sepolia.looksrare.org",
+    explorer: "https://sepolia.etherscan.io",
+    rpcUrl: "https://eth-sepolia.g.alchemy.com/v2",
+    baseApiUrl: "https://graphql-sepolia.looksrare.org",
+    osApiUrl: "https://testnets-api.opensea.io",
+    cdnUrl: "https://static-sepolia.looksnice.org",
+    rewardsSubgraphUrl: "https://api.thegraph.com/subgraphs/name/0xjurassicpunk/looks-distribution",
+    cloudinaryUrl: "https://looksrare.mo.cloudinary.net/sepolia",
+  },
```

Comment:
I feel like I missed something 😆 

Do we have all of this set up? On my end I'll need to spin up a Cloudinary delivery profile for the new cdn and a new Vercel project.